lightfm/evaluation.py

```
@@ -232,14 +235,14 @@ def auc_score(
 
     assert np.all(ranks.data >= 0)
```

Pylint (3.3.7)

[error] Undefined variable 'np'

(E0602)